### PR TITLE
Cigarette Radio Bugs

### PR DESCRIPTION
**Pull request by west3436**
The detective's cigarette box has been modified to be used as a speaker for covert cigarette radio bugs. Simply place the butt of the cigarette near your suspect and listen to their discussions via your cigarette box. Use the box in-hand to shut off the speaker to avoid giving your bug's location away. **New:** use the unlit cigarette in-hand prior to lighting to set a tag which will be shown on all messages received by the bug.

features

Idea from SkeletonSteve.
[gameplay]
[tested]

:cl:
 * rscadd: cigarette radio bugs now can be used to spy on unknowing suspects.
 * rscadd: adds cigarette radio bugs to the detective's gear box and Syndicate surveillance kit.

### CODE REVIEW

**Comment by bathosbathtime** [comment]
That's a pretty interesting feature. Good work on this one.

**Comment by gurfan** [comment]
Being able to label them like you can do with camera bugs would be nice.

**Comment by ghost** [comment]
why dont you just merge the functionality with camera bugs

**Comment by west3436** [comment]
> Being able to label them like you can do with camera bugs would be nice.

Working on this now


> why dont you just merge the functionality with camera bugs

Because this is more fun

File: code/game/objects/items/devices/radio/radio.dm [hunk 1]
```diff
@@ -622,3 +629,62 @@
 		playsound(loc, rustle_sound, 50, 1, -5)
 		return TRUE
 	return FALSE
+
+/obj/item/device/radio/phone/surveillance
```
Comment (Kurfursten) [inline code/game/objects/items/devices/radio/radio.dm:633]: Neat to see radio phones getting used

**Pull request merged**